pygmo/docstrings.cpp

```diff
@@ -1017,7 +1014,7 @@ The adaptation of the mutation variant is added to :class:`~pygmo.core.sade`
     variant_adptv (``int``): F and CR parameter adaptation scheme to be used (one of 1..2)
```

Comment:
Maybe ``F`` and ``CR`` should be bracketed by asterisks, as done a couple of lines below?